src/App.jsx

```diff
@@ -1,19 +1,31 @@
+import { useState } from "react";
 import { initialColors } from "./lib/colors";
 import Color from "./Components/Color/Color";
 import "./App.css";
+import ColorForm from "./Components/ColorForm/ColorForm";
 
 function App() {
+  const [colors, setColors] = useState(initialColors);
+
+  const handleAddColor = (color) => {
+    console.log("Adding color:", color);
+    setColors([...colors, color]);
+  };
+
   return (
     <> <ul>
       <h1>Theme Creator</h1>
-
-      <li className="color-container">
-        {initialColors.map((color) => {
-          return <Color key={color.id} color={color} />;
-        })}
-      </li>
+   
+      <ColorForm onAddColor={handleAddColor} />
+      <ul>
+        {colors.map((currentArrayItem) => (
+          <li key={currentArrayItem.id}>
+            <Color color={currentArrayItem} />
```

Comment:
maybe you want to consider renaming "currentArrayItem" to something more meaningful in this situation, something that describes a single item from the "colors" array.